auth/tokenizer.go

```diff
@@ -148,6 +149,10 @@ func (t oidcTokenizer) Validate(ctx context.Context, rawToken *oauth2.Token) (*v
 		return nil, err
 	}
 
+	if err := claims.Valid(); err != nil {
```

Comment:
Actually, we never checked if the claims are valid before building the v1.User. 
We only ever checked if the standard claims are valid in https://github.com/stackrox/infra-auth-lib/pull/2/files#diff-65179ac7f2ad5781639bcf7f4e91cdea6f92423de747e22aca4cf137203845d9R132.